internal/pkg/template/templates/workloads/partials/cf/taskrole.yml

```diff
@@ -3,13 +3,14 @@ TaskRole:
     'aws:copilot:description': 'An IAM role to control permissions for the containers in your tasks'
   Type: AWS::IAM::Role
   Properties:
-    {{- if or (.NestedStack) (eq .Observability.Tracing "AWSXRAY")}}
+    {{- $hasAddonPolicies := and .NestedStack (gt (len .NestedStack.PolicyOutputs) 0)}}
```

Comment:
will this result in a segfault if there are no addons since we're building w go1.17? 😢 I think w go1.18 it would shortcircuit and we'd be safe

Reply:
i'm crying rn😢

Reply:
just tested it, you're right. i'll fix :)